rimage/depth_map_raw.go

```diff
@@ -61,15 +62,16 @@ func ReadDepthMap(r io.Reader) (*DepthMap, error) {
 	}
 	switch firstBytes {
 	case MagicNumIntVersionX: // magic number for VERSIONX
-		return readDepthMapVersionX(r.(*bufio.Reader))
+		return readDepthMapVersionX(r)
```

Comment:
Yeah this is already a great change just from synchronizing the function signatures to all input a io.Reader. Love this.